logos/src/logos/main.py

```
@@ -1509,23 +1324,23 @@ async def start_pipeline():
+    _logosnode_facade = LogosNodeSchedulingDataFacade(_queue_mgr, None, runtime_registry=_logosnode_registry)
     _azure_facade = AzureSchedulingDataFacade(None)
 
     await _register_models_with_facades(_logosnode_facade, _azure_facade)
 
     model_registry = _build_model_registry()
 
-    # Scheduler: trust vLLM queue signals
-    scheduler = SimpleScheduler(
+    # Scheduler: use ETTFT-correcting scheduler (ablatable via env var)
+    ettft_enabled = os.getenv("LOGOS_SCHEDULER_ETTFT_ENABLED", "true").lower() == "true"
+    scheduler = ClassificationCorrectingScheduler(
```

Flake8 (7.3.0)

[error] 'ClassificationCorrectingScheduler' may be undefined, or defined from star imports: logos.dbutils.dbrequest

(F405)